tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/model/ESEmailAdressContactSearchTest.java

```diff
@@ -0,0 +1,28 @@
+package com.linagora.tmail.james.jmap.model;
+
+public class ESEmailAdressContactSearchTest implements EmailAddressContactSearchEngineContract{
```

Comment:
You likely gonna need to do some setup here with Elasticsearch, your factory to create the index, instantiating your EmailAddressContactSearchEngine, etc

Reply:
Yup, as I haven't known what exactly to setup so just let it be there to do it later.